x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -261,6 +262,24 @@ export const RulesList = ({
     setCurrentRuleToEdit(ruleItem);
   };
 
+  const onRunRule = async (rule: RuleTableItem) => {
+    try {
+      await runSoon({ http, id: rule.id });
+      toasts.addSuccess({
+        title: i18n.translate('xpack.triggersActionsUI.sections.rulesList.ableToRunRuleSoon', {
+          defaultMessage: 'Your rule is scheduled to run soon',
+        }),
+      });
+      await loadData();
```

Comment:
Not sure if reloading rules data makes sense here, probably the rule won't be executed yet :)

Reply:
Oh yes nice catch!

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/6299f9396aac9ded30dc345cde01abddfdf13ae6